src/lib/helpers.js

```diff
@@ -126,6 +126,11 @@ export const extractStringWithBrackets = (message) => {
   };
 };
 
+export function removeTextBetweenHashtags(input) {
```

Comment:
@tuseto 
There is a function called "clearButtonCodes" in the action-service.js that does the same, maybe we can use move it here in the helpers.js and remove "removeTextBetweenHashtags" this one.

Reply:
I moved the function into action-service, but cannot use the old one, they both are doing different things.

Reply:
@tuseto 

I was suppose to be the other way around, use the already existing function in the from the action service called (clearButtonCodes) and move it  into helpers.js and then remove the function you wrote.

Reply:
We can't clear all codes before the message is loaded (when all codes are cleared). It will result in more changes.